### ISSUE TITLE
Magellan ignores individual exposure times for channels

### ISSUE THREAD

**Issue by manuelmaeritz** [micro-manager/pycro-manager#696]
Dear developers,

does anyone else face this problem? When performing a Magellan acquisition from the GUI the exposure times which have been set in the 'Channels' menu are ignored for the acquisition. In the explore mode this seems to be working fine, though.

I am working on the latest nightly build of Micro-Manager 2.0.3 (20230912).

I would be grateful for any help!
Manuel

**Comment by henrypinkard** [comment]
This is very likely a bug in magellan. I'll take a look

**Comment by henrypinkard** [comment]
Im not able to reproduce this with the demo configuration. Are you using the latest nightly build? Are there specific acquisition settings that trigger this?

**Comment by manuelmaeritz** [comment]
Yes, I've updated to the nightly build 20230919 and the error persisted. On Monday I can send a screenshot of my acquisition settings. I played around a bit but did not see a particular setting triggering this behavior. I belief it is the acquisition time I have put for the Micro-Manager live view it is taking for the Magellan acquisition. I can confirm this on Monday too.
If it's of any help, this worked fine with an earlier version of Micro-Manager (maybe it was 2.0.0). I can not go back to this version though.
Here is the content of my configuration file: [config.txt](https://github.com/micro-manager/pycro-manager/files/12706318/config.txt)

**Comment by manuelmaeritz** [comment]
So when I put acquisition order to "Z, Channel", indeed the set exposure time is respected. However, for some reason the stage does not move in this configuration, so I can not use it. Also at the demo configuration the z-position does not change from image to image.
When the acquisition order is set to "Channel, Z" the stage moves but a uniform exposure time is taken for all the channels (see attached images for the acquisition settings). I saw the same behavior in the demo configuration.

What is more in the version 20230919 the file NDTiff.index file seem to be corrupted. When I try opening it with 
`data = Dataset(PATH)
data.as_array(stitched=True)`,
it is telling me `This is not a stitchable dataset`, while this works with data acquired with version 20230719 perfectly fine. Also the preview looks a bit strange, as if the image and the positions are wrong wired (see last image).


![OrderChZ_ChannelFluo](https://github.com/micro-manager/pycro-manager/assets/74652370/3cb33fb8-c4d1-4b23-9a75-8c5004b2c1be)
![OrderChZ_ChannelRhod](https://github.com/micro-manager/pycro-manager/assets/74652370/f29cb5cc-196f-44f8-9550-e554cce0c425)
![ChannelSettings](https://github.com/micro-manager/pycro-manager/assets/74652370/751ae6a9-f210-48f1-bf9b-ed4583a15f77)
![ChannelSettings](https://github.com/micro-manager/pycro-manager/assets/74652370/4729335b-f10a-4088-bca2-8ba48c335541)

**Comment by manuelmaeritz** [comment]
Thank you for the great support! I will test it as soon as possible and let you know.

> Is it possible your hard drive is full? I don't see this with the latest demo code. If you're able to reproduce this, could you collect a dataset with a very small image size (like 32x32) so and upload it so I can take a look and add it to the automated tests if theres a reading problem

In the meanwhile I saw this error appearing also with the older version, so it's got probably nothing to do with one of the latest nightly builds (maybe it was just bad luck when I saw it on 20230919). I belief I have enough space on the HD, but it's true I have the feeling it happens rather for large datasets, so maybe I reach some limits here. I will check this.
